python/cuml/ensemble/randomforest_common.pyx

```diff
@@ -343,26 +341,20 @@ class BaseRandomForestModel(Base):
                                         predict_proba=predict_proba)
         return preds
 
-    def _get_params(self, deep):
-        params = dict()
-        for key in BaseRandomForestModel.variables:
-            if key in ['handle']:
-                continue
-            var_value = getattr(self, key, None)
-            params[key] = var_value
-        return params
+    def get_param_names(self):
+        combined = super().get_param_names() + BaseRandomForestModel.variables
 
-    def _set_params(self, **params):
+        if ("handle" in combined):
```

Comment:
no parens needed in python style

Reply:
Is this in our coding style? I just prefer them since it makes multi-line if statements easier in python. So I now do it by default.

Reply:
Function has been removed so this is not needed.